nginx/config.go

```diff
@@ -308,6 +308,9 @@ http {
 				proxy_set_header X-Request-Id $request_id;
 				proxy_set_header X-Correlation-Id $correlation_id;
 				{{ end }}
+				{{ if and $routerConfig.RequestStartHeader (ne $appConfig.DisableRequestStartHeader true) }}
+				proxy_set_header X-Request-Start "t=${msec}";
+				{{end}}
```

Comment:
This line misses some spaces to conform with the surrounding style

Reply:
Fixed. Thanks! :+1: